spec/models/admin_user_spec.rb

```diff
@@ -1,3 +1,16 @@
-RSpec.describe AdminUser, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+require 'rails_helper'
+describe AdminUser, type: :model do
```

Comment:
as a side note, you may put an space between your `require` files and your actual code, for clarity reasons. If you think this is still clear, go ahead, it's up to your taste.